test/os.go

```diff
@@ -27,6 +28,46 @@ func TempDir(t testing.TB, base string) string {
 	return CanonPath(t, dir)
 }
 
+// IsNotExist calls os.Stat and asserts that the entry does not exist
+func IsNotExist(t testing.TB, dir, fname string) {
```

Comment:
nitpick:

```suggestion
// DoesNotExist calls os.Stat and asserts that the entry does not exist
func DoesNotExist(t testing.TB, dir, fname string) {
```

Reply:
I iterated though a couple of names on this one during development :) `IsNotExist` is not proper English, but it's in line with Go's `os.IsNotExist()` and `os.ErrNotExist`.

Reply:
Ah I was not aware of that, however the stdlib `os.IsNotExist(err error)` has a different use case i.e. it checks if the provided `err` is an `ErrNotExist` see https://cs.opensource.google/go/go/+/refs/tags/go1.21.4:src/os/error.go;l=90